Cargo.toml

```diff
@@ -39,7 +39,7 @@ name = "macrobenches"
 harness = false
 
 [features]
-default = []
+default = ["encoding"]
```

Comment:
I don't have very strong feelings about this, good arguments can be made either way.  The rationale is in the commit message.

I did some tests measuring binary size and performance with and without the feature and the differences were not especially large (around 5% for binary size, 10% for performance on my laptop).  That feels acceptable for a friendlier default especially considering those are "without including IO" numbers.